data/duke.txt

```diff
@@ -0,0 +1,5 @@
+E | Y | read book | 2020-04-04
```

Comment:
consider adding this to the .gitignore file! since this does not need to be tracked by version control